dll/win32/msi/custom.c

```diff
@@ -867,6 +867,17 @@ static UINT HANDLE_CustomType1( MSIPACKAGE *package, const WCHAR *source, const
     if (!(binary = get_temp_binary(package, source)))
         return ERROR_FUNCTION_FAILED;
 
+    #if defined(__REACTOS__) && defined(_M_AMD64)
+    {
+        DWORD arch;
+        get_binary_type(binary->tmpfile, &arch);
+        if (arch == SCS_32BIT_BINARY){
+            WARN("%s is a 32 bit custom action. Returning as ERROR_SUCCESS\n", debugstr_w( source ));
```

Comment:
Can you recall me, whether WARN also shows up in debug log during "normal" execution in debug mode?

Reply:
It is not shown in common debug execution, only shown under "set DEBUGPORT=+msi" execution.
I can do it always shown, but I am not sure if it's necessary. Since it will be reverted as soon as we have Wow64.
Example of what reports under VLC:
warn:(dll\win32\msi\custom.c:875) L"WixCA" is a 32 bit custom action. Returning as ERROR_SUCCESS

Reply:
I would tend to prefer having it shown whenever one installs an MSI on x64, unless it starts spamming the debug log a lot. This would help telling us when an MSI-packaged product would very certainly not work well afterwards on x64, because a certain number of 32-bit custom actions have been skipped.

Reply:
Yeah if you use `ERR` it should always show up, and that seems sane in a hack like this.

Reply:
Okkay, I will use ERR for this, thanks. :+1: